torch/csrc/jit/script/parser.cpp

```diff
@@ -650,7 +650,7 @@ struct ParserImpl {
         paramlist.range(), List<Param>(paramlist), return_annotation);
   }
 
-TreeRef parseClass() {
+  TreeRef parseClass() {
```

Comment:
nit: spurious whitespace, please revert

Reply:
Sorry about that. I reverted this change.